Client/Application/LeanplumIntegration.swift

```diff
@@ -245,4 +267,78 @@ class LeanplumIntegration {
         }
         return LeanplumSettings(environment: environment, appId: appId, key: key)
     }
+    
+    // This must be called before `Leanplum.start` in order to correctly setup
+    // custom message templates.
+    private func setupCustomTemplates() {
+        // These properties are exposed through the Leanplum web interface.
+        // Ref: https://github.com/Leanplum/Leanplum-iOS-Samples/blob/master/iOS_customMessageTemplates/iOS_customMessageTemplates/LPMessageTemplates.m
+        let args: [LPActionArg] = [
+            LPActionArg(named: LPMT_ARG_TITLE_TEXT, with: LPMT_DEFAULT_ASKT_TO_ASK_TITLE),
+            LPActionArg(named: LPMT_ARG_TITLE_COLOR, with: UIColor.black),
+            LPActionArg(named: LPMT_ARG_MESSAGE_TEXT, with: LPMT_DEFAULT_ASK_TO_ASK_MESSAGE),
+            LPActionArg(named: LPMT_ARG_MESSAGE_COLOR, with: UIColor.black),
+            LPActionArg(named: LPMT_ARG_ACCEPT_BUTTON_TEXT, with: LPMT_DEFAULT_OK_BUTTON_TEXT),
+            LPActionArg(named: LPMT_ARG_CANCEL_ACTION, withAction: nil),
+            LPActionArg(named: LPMT_ARG_CANCEL_BUTTON_TEXT, with: LPMT_DEFAULT_LATER_BUTTON_TEXT),
+            LPActionArg(named: LPMT_ARG_CANCEL_BUTTON_TEXT_COLOR, with: UIColor.gray)
+        ]
+        
+        let responder: LeanplumActionBlock = { (context) -> Bool in
+            guard let context = context else {
+                return false
+            }
+            
+            // Don't display permission screen if they have already allowed/disabled push permissions
+            if self.profile?.prefs.boolForKey(applicationDidRequestUserNotificationPermissionPrefKey) ?? false {
+                FxALoginHelper.sharedInstance.readyForSyncing()
+                return false
+            }
+            
+            // Present Alert View onto the current top view controller
+            let rootViewController = UIApplication.topViewController()
+            let title = NSLocalizedString(context.stringNamed(LPMT_ARG_TITLE_TEXT), comment: "")
+            let message = NSLocalizedString(context.stringNamed(LPMT_ARG_MESSAGE_TEXT), comment: "")
```

Comment:
I'm not sure `NSLocalizedString` works in the way. I'm not sure we're going to be using LeanPlum in any other local than `en-US`.

Reply:
I think this might be ok? I was trying to follow how their [example](https://github.com/Leanplum/Leanplum-iOS-Samples/blob/master/iOS_customMessageTemplates/iOS_customMessageTemplates/LPMessageTemplates.m#L192) was, but can remove if not necessary.